hcatalog/webhcat/svr/pom.xml

```diff
@@ -84,14 +84,14 @@
       </exclusions>
     </dependency>
     <dependency>
-      <groupId>com.sun.jersey.contribs</groupId>
-      <artifactId>wadl-resourcedoc-doclet</artifactId>
-      <exclusions>
-        <exclusion>
-          <groupId>com.sun.jersey</groupId>
-          <artifactId>jersey-server</artifactId>
-        </exclusion>
-      </exclusions>
+      <groupId>org.glassfish.jersey.ext</groupId>
```

Comment:
Why was this added, Could you explain this commit

Reply:
Hi @shivjha30
The ResourceDoclet from wadl-resourcedoc-doclet are incompatible with jdk-17 , so it is replaced by a compatible alternative: jersey-wadl-doclet .
Reason for incompatibility is that this ResourceDoclet relied on com.sun.javadoc APIs which was deprecated and finally removed in jdk13: [JDK-8215587](https://bugs.openjdk.org/browse/JDK-8215587)